sched/mqueue/mq_send.c

```diff
@@ -22,18 +22,18 @@
  * Included Files
  ****************************************************************************/
 
-#include  <nuttx/config.h>
-
-#include  <sys/types.h>
-#include  <mqueue.h>
-#include  <errno.h>
-#include  <debug.h>
-
-#include  <nuttx/irq.h>
-#include  <nuttx/arch.h>
-#include  <nuttx/cancelpt.h>
-
-#include  "mqueue/mqueue.h"
+#include <nuttx/config.h>
+#include <nuttx/arch.h>
+#include <nuttx/cancelpt.h>
+#include <nuttx/irq.h>
```

Comment:
In my opinion these (with the exception of `nuttx/config.h`) could be left in their current placement, after the C system headers.

But since the include order is not enforced by the NuttX coding standard, I won't block the PR for this.